packages/event-handler/src/appsync-graphql/RouteHandlerRegistry.ts

```diff
@@ -1,14 +1,15 @@
+import type { GenericLogger } from '@aws-lambda-powertools/commons/types';
 import type {
-  GenericLogger,
   RouteHandlerOptions,
   RouteHandlerRegistryOptions,
 } from '../types/appsync-graphql.js';
+import type { AppSyncGraphQLResolver } from './AppSyncGraphQLResolver.js';
 
 /**
  * Registry for storing route handlers for GraphQL resolvers in AWS AppSync GraphQL API's.
  *
  * This class should not be used directly unless you are implementing a custom router.
- * Instead, use the {@link Router} class, which is the recommended way to register routes.
+ * Instead, use the {@link AppSyncGraphQLResolver | `AppSyncGraphQLResolver`} class, which is the recommended way to register routes.
```

Comment:
Just for my own information, why is `AppSyncGraphQLResolver` repeated here? I see a similar pattern in a few other places in this PR and was just curious.

Reply:
The first argument after `@link` is the actual link/reference, the second is the label/description.

This is purely cosmetic, but writing it like that renders the link as `AppSyncGraphQLResolver` (monospaced) rather than plain text.